### PR TITLE
Add validator monitor and Alerts

### PR DESCRIPTION
**Pull request by unxnn**
https://github.com/stakewise/validator-monitor

### CODE REVIEW

File: charts/validators/values.yaml [hunk 1]
```diff
@@ -183,6 +183,37 @@ beaconChainRpcEndpointsRandomized: []
...
+## ref: https://github.com/stakewise/validator-monitor
+##
+validatorMonitor:
+  enabled: false
+  image:
+    repository: "europe-west4-docker.pkg.dev/stakewiselabs/public/validator-monitor"
+    tag: "v0.0.2"
+    pullPolicy: IfNotPresent
+  port: 9000
+  graphNodeUrl: https://api.thegraph.com/subgraphs/name/stakewise/stakewise-mainnet
```
Comment (tsudmi) [inline charts/validators/values.yaml:196]: lets add a comment here that for gnosis URL is different

File: charts/validators/values.yaml [hunk 2]
```diff
@@ -183,6 +183,37 @@ beaconChainRpcEndpointsRandomized: []
...
+  beaconNodeUrl: http://localhost:5052
+  indexChunckSize: 500
+  ## ETH=32; GNOSIS=16
+  ##
+  slotsPerEpoch: 32
+  operatorWallets:
```
Comment (tsudmi) [inline charts/validators/values.yaml:202]: This can be fetched from the subgraph.

File: charts/validators/templates/validator-monitor.yaml [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+            - containerPort: {{ .Values.validatorMonitor.port }}
+              name: metrics
+              protocol: TCP
+        {{- with .Values.validatorMonitor.resources }}
+          resources:
+            {{ toYaml . | nindent 12 | trim }}
+        {{- end }}
+
+---
+apiVersion: v1
```
Comment (tsudmi) [inline charts/validators/templates/validator-monitor.yaml:65]: Maybe it's better to split that into different files?

Comment (unxnn) [reply charts/validators/templates/validator-monitor.yaml]: Ok, agree, updated

**Review by tsudmi — APPROVED** [review]

**Pull request merged**